backend/dataall/modules/s3_datasets/cdk/dataset_stack.py

```diff
@@ -437,20 +449,68 @@ def __init__(self, scope, id, target_uri: str = None, **kwargs):
             },
         )
 
+        glue_sec_conf_enc_key = kms.Key(
+            self,
+            f'crwlr_log_enc_key_{dataset.GlueCrawlerName}',
+            removal_policy=RemovalPolicy.DESTROY,
+            alias=f'crwlr_log_enc_key_{dataset.GlueCrawlerName}',
```

Comment:
- can we have both `alias` and `kms.Key` construct id be  `{dataset.GlueCrawlerName}-log-enc-key` where dataset_basename is: 
```
        dataset_basename = NamingConventionService(
            target_uri=dataset.datasetUri,
            target_label=dataset.label,
            pattern=NamingConventionPattern.GLUE_ETL,
            resource_prefix=env.resourcePrefix,
        ).build_compliant_name()
```


    - that way the naming convention is prefixed with `resource_prefix`
    - also no use of `crawler` in the name since we are applying to both crawler and glue job


- nit: we can also set it once as variable and pass that as both construct id and alias name

Reply:
done